cypress/e2e/accessibility.cy.ts

```diff
@@ -1,63 +1,71 @@
 describe("Accessibility tests in light mode", () => {
     beforeEach(() => {
         cy.login();
-        cy.get(".light-button").click();
     });
 
     it("Checks clients page", () => {
         cy.visit("/clients");
 
+        cy.get(".light-button").click();
         cy.checkAccessibility();
     });
 
     it("Checks lists page", () => {
         cy.visit("/lists");
 
+        cy.get(".light-button").click();
         cy.checkAccessibility();
     });
 
     it("Checks calendar page", () => {
         cy.visit("/calendar");
 
+        cy.get(".light-button").click();
+        cy.get("table", { timeout: 5000 }).should("be.visible");
```

Comment:
The calendar takes time to be loaded. Without this line, the test will check the loading page accessibility rather than the loaded calendar (the calendar has a table so we are waiting for the calendar to be loaded essentially)

5 seconds is kinda reasonable as it takes a few second for the page to be fully loaded (we also don't want the test to fail because of slow internet)

Reply:
`cy.get` will retry for up to 4s iirc, is this not enough? Can we remove `timeout` and still get a passing test?

Reply:
Yep we did what Tom has suggested instead.

cy.wait doesn't seem to work, as when we use cy.intercept it is checking whether the calendar page is loaded, but not the calendar component itself. So even when it loaded an empty page with just the heading and no calendar, it still returns true and run the accessibility test on the empty page